app/templates/items/detail/tabs/_item_detail.hbs

```diff
@@ -78,6 +78,16 @@
           </div>
         </div>
       </div>
+      <div class="row detail-section">
+        <div class="columns small-12">
+          <div class="row">
+            <div class="columns small-3">Saleable </div>
+            <div class="columns small-9 value description">
+              {{input class="checkbox-item-details" type="checkbox" id="saleable" checked=item.saleable }}
```

Comment:
It's error-prone to have IDs on elements, especially in a reusable partial. I already see the same ID being used twice in this PR. As a best-practice we want to avoid them. And they're rarely very useful anyways :)

Reply:
sure. Will remove the ids. Thanks @patrixr